next-sitemap.config.js

```diff
@@ -28,29 +28,40 @@ module.exports = {
 		'sitemap.xml',
 		'/server-sitemap.xml',
```

Comment:
Might not be in the scope of this PR, but wanted to note that I don't see the product landing paths on this list. We should make sure to add those somewhere. i.e. /terraform, /vault, /consul etc.

Reply:
The landing pages are generated with next-sitemap and are included in the `sitemap-0.xml` file. The `sitemap.xml` file links all sitemaps for crawlers